### ISSUE TITLE
Migrate an existing angular app into a mfe host

### ISSUE THREAD

**Issue by timtos** [nrwl/nx#10496]
## Current Behavior
I have an existing angular app using the latest version of nx (14.1.9). I now want to migrate this relatively big app to be a mfe host. In a first step even without any remotes. The current app should just function as a host. So I called the `ng generate @nrwl/angular:setup-mfe` schematic to transform the current app into a host:

![image](https://user-images.githubusercontent.com/2648014/170767142-a671f10c-04b6-459a-a156-d083f2168778.png)

Everything compiles and the application can be served but when I reload the page in the browser, I get:

![image](https://user-images.githubusercontent.com/2648014/170767440-fc59bea6-72a1-495e-817f-bab6968e44b3.png)

I couldn't find any documentation on how to migrate an existing app to a mfe host and I have absolutely no idea where to even look for a solution or how to tackle this error. I trimmed down the app so that the main.ts and the app component are very simple but I cannot get it to work. Is it some configuration issue? Is my assumption correct that this should work without any issues?

## Expected Behavior
I was expecting, that adding a host is not a problem as it is just the host that can contain the "old" app without any issues.

### Failure Logs
```
Uncaught Error: Shared module is not available for eager consumption: 482323
    at Object.__webpack_require__.m.<computed> (consumes:287:1)
    at __webpack_require__ (bootstrap:24:1)
    at fn (hot module replacement:61:1)
    at Module.31126 (polyfills.js:14:80)
    at __webpack_require__ (bootstrap:24:1)
    at startup:6:1
```

### Environment
Node : 16.13.2
OS : win32 x64
yarn : 1.22.5

**Comment by leosvelperez** [comment]
Thanks for reporting this!

Could you please share a repro? Maybe the trimmed-down app you mentioned.

**Comment by timtos** [comment]
@leosvelperez Thank you for your help. I doube-checked what you asked for and everything looks good. I'll try to trim down the app; although, it could be tough, as it is a kinda huge app ;)

**Comment by timtos** [comment]
@leosvelperez It seems like I was able to narrow it down and that angular i18n is making trouble; especially the line `'@angular/localize/init';`

Sample repo can be found [here](https://github.com/timtos/mfe-test-workspace).
If one removes the line import `'@angular/localize/init';` in the polyfills.ts the message disappears. 

Steps that were taken to create the repo:
- npx create-nx-workspace@latest
- nx migrate latest
- npx nx migrate --run-migrations
- yarn
- yarn add -W @angular/localize 
- yarn nx g @angular/localize:ng-add

What am I doing wrong?

P.S.: One little side question: Why is `npx create-nx-workspace@latest` not creating a workspace with the latest nx version? I had to call `nx migrate latest` explicitly.
